### PR TITLE
Incorperate the character design function into the rest of game flow

### PR DESCRIPTION
**Pull request by lattai**

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -352,3 +352,29 @@ a:hover {
...
+  height : 185px;
+}
+
+#pixel-body {
+  position: absolute;
+  width : 260px;
+  left : 10px;
+  top : 160px;
+  height : 260px;
+
```
Comment (menynexi) [inline src/main/webapp/style.css:372]: extra line

File: src/main/webapp/gameStage.html [hunk 2]
```diff
@@ -23,5 +23,8 @@ <h2 class="step-outline">Welcome!</h2>
         <button class="step-button" onclick="getDialogue()" name="nextButton">NEXT</button>
       </spam>
   </div>
+  <div id = 'image-container'>
+      <img id = "pixel-body" src="images/studentBody.png" alt="pixel you">
```
Comment (menynexi) [inline src/main/webapp/gameStage.html:27]: maybe a better name would be "pixel avatar" ?

File: src/main/webapp/uploaded.html [hunk 3]
```diff
@@ -9,7 +9,9 @@
   <body onload = "getImage()">
     <h1>Uploaded</h1>
     </br>
-    <div id = 'image-container'></div>
+    <div id = 'image-container'>
+        <img id = "pixel-body" src="images/studentBody.png" alt="pixel you">
+    </div>
     <div id = 'nickname-container'></div>
```
Comment (menynexi) [inline src/main/webapp/uploaded.html:16]: extra line

File: src/main/webapp/gameDialogue.js [hunk 4]
```diff
@@ -25,3 +25,30 @@ function addDialogueToDom(dialogue) {
   const quoteContainer = document.getElementById('dialogue-container');
   quoteContainer.innerText = dialogue;
 }
+
+
+function getImage() {  
+  fetch("/image-handler")
+      .then(response => response.text())
+      .then(message => {
+          var messageArray = message.split("\n");
```
Comment (lzoebaker) [inline src/main/webapp/gameDialogue.js:34]: Maybe add a comment about what format the messageArray comes in/ what each line represents.

**Pull request merged**